test/extended/images/mongodb_ephemeral.go

```diff
@@ -24,8 +24,13 @@ var _ = g.Describe("[images][mongodb] openshift mongodb image", func() {
 			g.By("creating a new app")
 			o.Expect(oc.Run("new-app").Args("-f", templatePath).Execute()).Should(o.Succeed())
 
-			g.By("expecting the mongodb service get endpoints")
-			o.Expect(oc.KubeFramework().WaitForAnEndpoint("mongodb")).Should(o.Succeed())
+			g.By("waiting for the deployment to complete")
+			exutil.WaitForADeployment(
+				oc.KubeREST().ReplicationControllers(oc.Namespace()),
```

Comment:
> I believe that function call will translate to oc project or something like that?

It turns out that namespace is saved in the struct and automatically updated in `beforeEach()`/`afterEach()`. 2 invocation isn't expensive.